docs/examples/partitioning/README.md

```diff
@@ -0,0 +1,82 @@
+PETL - Partitioning Example
+==========================
```

Comment:
Probably not worth redoing at this point, but might make sense to put the whole partitioning example in a separate commit.